src/components/UserDashboard/UserDashboard.module.css

```diff
@@ -1,3 +1,8 @@
+.dashboadRowContainer {
+  margin: 0 !important;
```

Comment:
I know `!important` is valid css. But it's use should be the last-last resort when there is no other possibilities.

What did you tried to force the use of it? Can't you make your selector more specific?